packages/icons-react/src/animated-icons/HamburgerCloseAnimated.tsx

```diff
@@ -6,6 +6,7 @@ import { variants } from "../icons/types";
 interface Props {
     isBurger: boolean;
     variant?: variants;
+    title?: [string, string];
```

Comment:
I stedet for en string array, kan vi bruke hamburgerTitle og closeTitle?

Reply:
sure